### DISCUSSION TITLE
euphony-listener is an error that occurs in Android Studio 2021.1.1 version

### DISCUSSION THREAD

**Discussion by sunyeongan** [euphony-io/euphony#162] [General]
Hello! I installed the Android Studio 2021.1.(chipmunk) 1 version on my new computer and tried to build the euphony-listener, and there was a collision problem (probably) with the gradle version.

The error in my computer's Android studio is as follows
1.
Can't determine type for tag '<macro name="m3_comp_bottom_app_bar_container_color">?attr/colorSurface</macro>'
values.xml
2.
"No toolchains found in the NDK toolchains folder for ABI with prefix: mips64el-linux-android"

**euphony-listener content was:** 
bulid.gradle(Project)
classpath 'com.android.tools.build:gradle:3.5.2'

bulid.gradle (Module)
android{
compileSdkVersion 29
}
defaultConfig{
targetSdkVersion 29
}
gradle-wrapper.properties
distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip

**The build was successful after updating the gradle version**
**The correction I made is as follows.**

Modify from bulid.gradle(Project)
classpath 'com.android.tools.build:gradle:7.2.1'

Modifying from bulid.gradle (Module)
android{
compileSdkVersion 32
}
defaultConfig{
targetSdkVersion 32
}
Modifying from gradle-wrapper.properties
distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip

I hope it will be reflected in the euphony-listener

**Comment by yeonns** [comment]
@sunyeongan Hi! Thank you for reporting this issue :)
I tried to reproduce your issue, but I have no errors and build of euphony-listener is success. 😢 
Is there anyone who can reproduce this issue? 👀 

My dev environment is:
- **Android Studio Version**: Android Studio Chipmunk | 2021.2.1 Patch 1  

![image](https://user-images.githubusercontent.com/10149398/180462316-1cf021d7-023f-4f2a-a3b7-28242d890e41.png)  
- **OS**: Windows 11
- **euphony-listener status**
```sh
$ git log -5 --oneline
c02317f (HEAD -> master, origin/master, origin/HEAD) Modify Indentation for README.md and README_KR.md (#6)
```
- **build logs**
```
Executing tasks: [:app:assembleDebug] in project C:\git\openup\euphony-listener

> Task :app:preBuild UP-TO-DATE
> Task :app:preDebugBuild UP-TO-DATE
> Task :app:checkDebugManifest
> Task :app:generateDebugBuildConfig
> Task :app:mainApkListPersistenceDebug
> Task :app:javaPreCompileDebug
> Task :app:compileDebugRenderscript NO-SOURCE
> Task :app:compileDebugAidl NO-SOURCE
> Task :app:generateDebugResValues
> Task :app:generateDebugResources
> Task :app:createDebugCompatibleScreenManifests
> Task :app:processDebugManifest
> Task :app:mergeDebugResources
> Task :app:processDebugResources
> Task :app:compileDebugJavaWithJavac
> Task :app:compileDebugSources
> Task :app:mergeDebugShaders
> Task :app:compileDebugShaders
> Task :app:generateDebugAssets
> Task :app:mergeDebugAssets
> Task :app:processDebugJavaRes NO-SOURCE
> Task :app:checkDebugDuplicateClasses
> Task :app:transformClassesWithDexBuilderForDebug
> Task :app:validateSigningDebug
> Task :app:signingConfigWriterDebug
> Task :app:mergeDebugJavaResource
> Task :app:mergeExtDexDebug
> Task :app:mergeDebugJniLibFolders
> Task :app:mergeDebugNativeLibs
> Task :app:stripDebugDebugSymbols
> Task :app:mergeDexDebug
> Task :app:packageDebug
> Task :app:assembleDebug

BUILD SUCCESSFUL in 40s
24 actionable tasks: 24 executed
```

**Comment by zion830** [comment]
* I tried, but I also have no problem to build
* I modified gradle versions as you wrote, and I got an error because `android:exported` is mandatory after android 12.
* Other than this, both work well

### My build environment
```
Android Studio Chipmunk | 2021.2.1 Patch 1
Build #AI-212.5712.43.2112.8609683, built on May 19, 2022
Runtime version: 11.0.12+0-b1504.28-7817840 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.2.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Registry: external.system.auto.import.disabled=true, ide.balloon.shadow.size=0
Non-Bundled Plugins: Key Promoter X (2021.3), wu.seal.tool.jsontokotlin (3.7.4), com.intellij.marketplace (212.5712.51), Dart (212.5744), mobi.hsz.idea.gitignore (4.3.0), org.jetbrains.kotlin (212-1.7.10-release-333-AS5457.46), com.chrisrm.idea.MaterialThemeUI (6.9.1), io.flutter (69.0.2), com.mallowigi (50.2.0)
```

**Comment by sunyeongan** [comment]
@zion830 @yeonns oh... Then it's a different matter! Thank you for your kind answer. And from now on, I will write down the contents of the development environment at a glance. It looks very nice.
